spec/Auth.spec.js

```diff
@@ -106,6 +106,8 @@ describe('Auth', () => {
         updatedAt: updatedAt.toISOString(),
       }
     );
+    Parse.Server.cacheController.clear();
```

Comment:
Should we convert this to async while already at it? Would that be a lot of work? Then we could also remove the 1s sleep below.

Reply:
The delay is just to make sure enough time has passed to make sure the delay in the cache has expired